apps-rendering/src/item.ts

```diff
@@ -65,6 +66,7 @@ interface Fields extends ArticleFormat {
 	relatedContent: Option<ResizedRelatedContent>;
 	logo: Option<Logo>;
 	webUrl: string;
+	promotedNewsletter?: Newsletter;
```

Comment:
Can I suggest using an `Option` for this (see the other fields above for examples)? We try to avoid use of optional properties (`A | undefined`) in AR.

```suggestion
	promotedNewsletter: Option<Newsletter>;
```